Scripts/deploy-docker-databases.sh

```diff
@@ -0,0 +1,48 @@
+cd ..
+mkdir ContainerDB_persistence_files
+cd  ContainerDB_persistence_files
+mkdir if_data if_config if_log ts_data ts_log ts_config pg_config pg_log pg_data ch_log ch_config ch_data
+
+docker run -d   -p 8087:8086 \
```

Comment:
only test one running database at a time.
If you are running the other database servers when you are benchmarking one, it will harm the performance of the one under benchmarking (competition for CPU and memory caused by running services and background jobs e.g. indexing and compression jobs).